client/my-app/src/components/ExpenseItem/ExpenseItem.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState, Dispatch, SetStateAction, FC } from "react";
+import { removeExpense } from "../../api/expenses";
+import { ExpenseModal } from "../ExpenseModal/ExpenseModal";
+
+type Props = {
+  currentExpense: Expense,
+  setExpenses: Dispatch<SetStateAction<Expense[]>>,
+  expenses: Expense[],
+};
+
+export const ExpenseItem: FC<Props> = ({
+  currentExpense,
+  setExpenses,
+  expenses
+}) => {
+  const [isUpdating, setIsUpdating] = useState(false);
+
+  const handleDelete = async() => {
+    try {
+      await removeExpense(currentExpense.id);
+    } catch (err: any) {
+      throw new Error(err);
```

Comment:
and here - this is useless
you need handle error somehow and show info to user (notification or smth else)